configs/components/openssl-1.1.0.rb

```diff
@@ -0,0 +1,145 @@
+component 'openssl' do |pkg, settings, platform|
```

Comment:
There's some common stuff here w/ the other openssl.rb file. Least the ENVIRONMENT, FLAGS, TARGETS + INSTALL STEPS. Could there be a `_base_openssl.rb` component?